src/client/app/controllers/controllers.js

```diff
@@ -0,0 +1,48 @@
+interview_coding_app.controller('show_fav_stuff_list', function ($scope, FavStuffService) {
```

Comment:
`FavStuffService`'s dependencies are annotated but controller's aren't.

Also, this one has a `different_case`